scripts/kubernetes_node/create.py

```diff
@@ -64,7 +64,7 @@ def execute_command(command):
 if __name__ == '__main__':
 
     # Check if Docker PS works
-    docker = check_command('docker ps')
+    docker = check_command('sudo docker ps')
```

Comment:
This should not be necessary since we should already have the agent user in the docker group.

Reply:
@EarthmanT If that the case then we can remove this branch